package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "scripts": {
+    "darwin": "cd darwin && npm install && cd ..",
+    "linux": "cd linux && npm install && cd ..",
+    "windows": "cd windows && npm install && cd ..",
+    "lint": "eslint ."
+  },
+  "devDependencies": {
+    "babel-eslint": "^7.2.3",
+    "eslint": "^4.7.2",
+    "eslint-config-airbnb": "^15.1.0",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-jsx-a11y": "^5.1.1",
+    "eslint-plugin-react": "^7.4.0"
```

Comment:
```
"eslint-plugin-jsx-a11y": "^5.1.1",
"eslint-plugin-react": "^7.4.0"
```
are actually not needed

Reply:
We can use `eslint-config-airbnb-base` wchich doesn't depend on those two packages